build/getIUCN.js

```
@@ -0,0 +1,93 @@
+        variables: { sname: sname },
+        type: "warning",
+      });
+      STATE.IUCNcache[sname] = {
+        scopes: [{ scope: "Global", status: "NA", url: null }],
+      };
+      updatePrefs("IUCNcache.json", STATE.IUCNcache);
+      return true;
+    }
+    console.error("Error fetching IUCN data:", error.message);
+    return false;
+  }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)